pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -109,6 +109,11 @@ type PostgresClusterSpec struct {
 	// +optional
 	Patroni *PatroniSpec `json:"patroni,omitempty"`
 
+	// Suspends the rollout and reconciliation of changes made to the
+	// PostgresCluster spec.
+	// +optional
+	Pause *bool `json:"pause,omitempty"`
```

Comment:
Looked at the story again, `paused` is what we discussed. I'm trying to think how I got to `pause` here and I think I started looking at how we implemented `shutdown` and started to think of this behavior as more of a verb rather than state. Funny enough, `shutdown` is both.... :dizzy_face: In any case, I'll update this to `paused`, thanks for the catch.